File: src/main/java/dansplugins/radios/Radios.java
package dansplugins.radios;

import org.bukkit.plugin.java.JavaPlugin;

public final class Radios extends JavaPlugin {

    @Override
    public void onEnable() {
        getLogger().info("Radios " + getDescription().getVersion() + " enabled.");
    }

    @Override
    public void onDisable() {
        getLogger().info("Radios disabled.");
    }
}

File: src/main/kotlin/com/dansplugins/radios/Radios.kt
package com.dansplugins.radios

import org.bukkit.plugin.java.JavaPlugin

class Radios : JavaPlugin() {

    override fun onEnable() {
        logger.info("Radios ${description.version} enabled.")
    }

    override fun onDisable() {
        logger.info("Radios disabled.")
    }
}
